src/api/json/catalog.json

```diff
@@ -3204,8 +3204,8 @@
     {
       "name": "Google Web Designer Custom Component Manifest",
       "description": "Schema for Custom Component Manifest used by Google Web Designer",
-      "fileMatch": ["manifest.json"],
-      "url": "https://raw.githubusercontent.com/Waterloo/WebComponent-Manifest/main/src/gwd-component-manifest/gwd-manifest-schema.json"
+      "fileMatch": ["*manifest.json"],
```

Comment:
manifest.json is a very generic file name. Can you make it more precise to avoid false positives for IDEs/editors in doing correct file matching?